docs/language_specification/statements/instructions/unitary_instructions.md

```diff
@@ -0,0 +1,188 @@
+Unitary instructions include:
+
+- [named gates](unitary_instructions.md#named-gates), and
+- compositions of [gate modifiers](unitary_instructions.md#gate-modifiers) acting on a named gate.
+
+Note that a composition of gate modifiers acting on a named gate is itself a gate. 
+
+??? info "Grammar for unitary instructions"
+    
+    _unitary-instruction_:  
+    &emsp; _gate_ _qubit-arguments_  
+
+    _gate_:  
+    &emsp; _inv-gate_  
+    &emsp; _pow-gate_  
+    &emsp; _ctrl-gate_  
+    &emsp; _named-gate_  
+
+    _inv-gate_:  
+    &emsp; `inv.`gate
+    
+    _pow-gate_:  
+    &emsp; `pow(`_parameter_`).`_gate_
+    
+    _ctrl-gate_:  
+    &emsp; `ctrl.`_gate_
+
+    _named-gate_:  
+    &emsp; _identifier_ _parameters_~opt~  
+
+    _parameters_:  
+    &emsp; __`(`__ _parameter-list_ __`)`__
+
+    _parameter-list_:  
+    &emsp; _parameter_  
+    &emsp; _parameter-list_ __`,`__ _parameter_
+
+    _parameter_:  
+    &emsp; _integer-literal_  
+    &emsp; _floating-literal_
+
+    _qubit-arguments_:  
+    &emsp; _qubit-argument-list_
+
+    _qubit-argument-list_:  
+    &emsp; _qubit-argument_  
+    &emsp; _qubit-argument-list_ __`,`__ _qubit-argument_
+
+    _qubit-argument_:  
+    &emsp; _qubit-variable_  
+    &emsp; _qubit-index_
+
+    _qubit-variable_:  
+    &emsp; _identifier_
+
+    _qubit-index_:  
+    &emsp; _index_
+
+## Named gates
+
+Named gates define single-qubit or multi-qubit unitary operations
+that change the state of a qubit register in a deterministic fashion.
```

Comment:
This description applies to unitary instructions in general and should then be moved above (see other suggestion)